src/sentry/static/sentry/app/components/group/releaseStats.tsx

```diff
@@ -13,15 +13,19 @@ type Props = {
   environments: Environment[];
   allEnvironments: Group | undefined;
   group: Group | undefined;
+  currentRelease: any | null | undefined; // TODO(ts)
```

Comment:
Can we go ahead and type this since you're actively working/testing it?

Reply:
I'll follow up in a separate PR for this.

Reply:
Updated with types in this PR.